go/common/tribool/tribool.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package tribool
+
+// Tribool is a three-valued logic type.
+// It can be either True, False or Unknown.
+// It is useful for representing the result of a comparison
+// when the comparison is not possible or could not be determined.
+type Tribool struct {
```

Comment:
The type should implement `Stringer` to support debug printing.

Reply:
right, will add